dbdimp.c

```diff
@@ -2083,11 +2081,33 @@ MYSQL *mysql_dr_connect(
 
 	    mysql_ssl_set(sock, client_key, client_cert, ca_file,
 			  ca_path, cipher);
-#if MYSQL_VERSION_ID >= SSL_VERIFY_VERSION && MYSQL_VERSION_ID <= SSL_LAST_VERIFY_VERSION
+#ifdef MYSQL_SSL_MODE
+            if (ssl_verify_true)
+              ssl_mode = SSL_MODE_VERIFY_IDENTITY;
+            else if (ca_file && (strlen(ca_file) > 0))
```

Comment:
Do we need check for `strlen()`? I have not find any information about it and variables are already passed to `mysql_ssl_set()` without `strlen()` check... So I'm not sure there...